profile/README.md

```diff
@@ -1,3 +1,16 @@
 # Artefactual
 
 For more information about us, please [visit artefactual.com](https://www.artefactual.com/).
+
+## Contributing
+
+We are actively developing the [Contributors portal](https://contributors.artefactual.com) to host guidelines and information
+related to contributing to AtoM, Archivematica, and the dependencies which we maintain.
+
+## Related organizations
+
+Most of our projects are hosted at one of the following:
+
+- [Main Artefactual repo](https://www.github.com/artefactual), this page
+- [Artefactual labs](https://www.github.com/artefactual-labs), smaller projects and experiments
+- [Archivematica repo](https://www.github.com/archivematica), for related issues and some projects
```

Comment:
if you wanted you could add [Artefactual-SDPS](https://github.com/artefactual-sdps) for projects related to Enduro